datasource/hcp-packer-image/data.go

```diff
@@ -26,8 +29,22 @@ type Config struct {
 	common.PackerConfig `mapstructure:",squash"`
 	// The name of the bucket your image is in.
 	Bucket string `mapstructure:"bucket_name" required:"true"`
+	// The name of the channel to use when retrieving your image.
+	//
+	// If using several images from a single iteration, you may prefer
+	// sourcing an iteration first, and referencing it for subsequent uses,
+	// as getting the iteration induces a potentially billable request.
+	//
+	// Either this or `iteration_id` MUST be set.
+	//
+	// Mutually exclusive with `iteration_id`
+	Channel string `mapstructure:"channel" required:"false"`
```

Comment:
Packer let's you set these two as required and at the same time let them not be set. The validation of those required fields is just the one you implemented. I've seen other plugins with mutually exclusive configs that were required.